components/CrawlUrls.tsx

```
@@ -0,0 +1,254 @@
+      return { variant: 'default', className: 'bg-yellow-400 text-black hover:bg-yellow-500' };
+    case 'crawling':
+       // Blue background, white text (default variant might be blue)
+      return { variant: 'default', className: 'bg-blue-600 text-white' };
+    case 'completed':
+       // Green background, white text
+      return { variant: 'default', className: 'bg-green-600 text-white hover:bg-green-700' };
+    case 'crawl_error':
+    case 'discovery_error':
+       // Red background, white text (destructive variant might be red)
+      return { variant: 'destructive', className: 'bg-red-600 text-white' };
+    case 'discovering':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)